e2e/template_e2e_tests/run_e2e_test.sh

```diff
@@ -61,27 +61,29 @@ done
 # The following vars need to be set in the environment when runnin the e2e tests.
 # For example, set up env vars and action secrets in Github Action workflow.
 echo "ORGANIZATION_ID=$ORGANIZATION_ID"
-echo "FOLDER_ID=$FOLDER_ID"
 echo "BILLING_ACCOUNT=$BILLING_ACCOUNT"
+echo "FOLDER_ID=$FOLDER_ID"
```

Comment:
What should be FOLDER_ID set to?

Reply:
The FOLDER_ID is optional now since we no longer able to use the Joonix org. Let me remove it for now.